examples/js/loaders/OBJLoader2.js

```diff
@@ -0,0 +1,996 @@
+/**
+  * @author Kai Salmen / https://kaisalmen.de
+  * Development repository: https://github.com/kaisalmen/WWOBJLoader
+  */
+
+'use strict';
+
+if ( THREE.OBJLoader2 === undefined ) { THREE.OBJLoader2 = {} }
+THREE.OBJLoader2.version = '1.0.1';
+
+/**
+ * Use this class to load OBJ data from files or to parse OBJ data from arraybuffer or text
+ * @class
+ *
+ * @param {THREE.DefaultLoadingManager} [manager] Extension of {@link THREE.DefaultLoadingManager}
+ */
+THREE.OBJLoader2 = (function () {
+
+	function OBJLoader2( manager ) {
+		this.manager = ( manager == null ) ? THREE.DefaultLoadingManager : manager;
+
+		this.path = '';
+		this.fileLoader = new THREE.FileLoader( this.manager );
+
+		this.meshCreator = new THREE.OBJLoader2.MeshCreator();
+		this.parser = new THREE.OBJLoader2.Parser( this.meshCreator );
```

Comment:
Done! I have pushed V1.0.3.
Uglified/Minified version is built with mangle=false otherwise it becomes almost impossible to predict the names of internally renamed objects when building the code string.